tensorflow/python/framework/test_util.py

```diff
@@ -1014,8 +1014,6 @@ def prepare_config(config):
       config.graph_options.optimizer_options.opt_level = -1
       config.graph_options.rewrite_options.constant_folding = (
           rewriter_config_pb2.RewriterConfig.OFF)
-      config.graph_options.rewrite_options.arithmetic_optimization = (
```

Comment:
@yifeif can you confirm if this is intended?

Reply:
I think we should keep that. Updated! Thanks!